src/main/java/org/opensearch/neuralsearch/processor/factory/TextChunkingProcessorFactory.java

```diff
@@ -45,6 +48,16 @@ public TextChunkingProcessor create(
     ) throws Exception {
         Map<String, Object> fieldMap = readMap(TYPE, processorTag, config, FIELD_MAP_FIELD);
         Map<String, Object> algorithmMap = readMap(TYPE, processorTag, config, ALGORITHM_FIELD);
-        return new TextChunkingProcessor(processorTag, description, fieldMap, algorithmMap, environment, clusterService, analysisRegistry);
+        boolean ignoreMissing = readBooleanProperty(TYPE, processorTag, config, IGNORE_MISSING, DEFAULT_IGNORE_MISSING);
+        return new TextChunkingProcessor(
```

Comment:
How about applying builder pattern and create a DTO object  to form TextChunkingProcessor Object? Currently, argument size in the TextChunkingProcessor looks too long.

Thoughts @martin-gaievski ?

Reply:
We can do this in a separate PR aiming refactoring of plugin classes, for scope of this PR I think we're good with adding just a field.

Reply:
discussed offline, there are no more concerns